index.js

```diff
@@ -149,12 +149,24 @@ File.isVinyl = function(file) {
 // Or stuff with extra logic
 Object.defineProperty(File.prototype, 'contents', {
   get: function() {
+    if (this.stat && this.stat.atime) {
+      this.stat.atime.setTime(Date.now());
```

Comment:
why setTime instead of an assignment?  We aren't sure if the value is actually a date object, since a vinyl object could be constructed with an int for timestamp (and it should work fine, I think)